data/human/human missions.txt

```diff
@@ -4925,3 +4925,146 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+
+
+mission "Syndicate Business 1"
+	name "Rescue Kidnapped Man in <waypoints>"
+	description "A mysterious man has been kidnapped by a heavy pirate warship in <waypoints>. Destroy the ship to rescue him."
+	minor
+	to offer
+		random < 40
+		credits > 5000000
+	source 
+		government Syndicate
+		attributes "core"
+	waypoint "Moktar"
+	destination "Sunracer"
+	on offer
+		conversation
+			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words "EMERGENCY DISTRESS SIGNAL" flash in red. A terrified-looking man in a suit appears on the screen. "I don't have much time - I've been kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape pod."` 
+			`You hear banging and muffled shouting over the intercom. "I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-" The message cuts out.`
+			choice
+				`	(Assist the mysterious man.)`
+					goto assist
+				`	(Don't assist the man.)`
+					decline
+			label assist
+			`	You note that the distress signal came from a warship in <waypoints> and wonder why a Syndicate employee would be kidnapped by pirates. Anyway, time to get ready for a fight.`
+				accept
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+		conversation
+			`When you enter <waypoints>, your sensors detect a pirate Vanguard, but no sign of any escape pod. Your ship is immediately hailed, and the pirate opens communication.`
+			`	The voice of the captain of the Golden Jewel shouts out towards you. "You were unwise to get involved in our business. Hand over that pompous salesman Howard, and I'll let you live."`
+			choice
+				`	"Uhhh... hand over who?"`
+					goto who
+				`	"I don't negotiate with pirates!"`
+					goto negotiate
+				`	"You can take him, but I don't know where he is."`
+					goto who
+			label who
+			`	"Bah, you lying buffoon! I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you!"`
+				goto power
+			label power
+			`	The Vanguard powers up its weapons.`
+	npc kill
+		government "Independent (Killable)"
+		personality heroic vindictive staying
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"		
+		conversation 
+			`The <npc> has been destroyed. You receive a hail from a small escape pod. "Thank you, Captain, you've saved my life. I'm Howard. Meet me on <destination> to receive your reward."`
```

Comment:
But how does he get to Sunracer?

Reply:
I think we could say you collect the escape pod? Or maybe make you board him?